### ISSUE TITLE
The getter 'WidgetState' isn't defined for the class '_HoverToggleState'

### ISSUE THREAD

**Issue by pishguy** [iampawan/VelocityX#210]
why i get this error on older versions of flutter?

```
Error (Xcode): ../../../../../../.pub-cache/hosted/pub.dev/velocity_x-4.2.0/lib/src/flutter/hover.dart:31:42: Error: The getter 'WidgetState' isn't defined for the class '_HoverToggleState'.
```

**Comment by admlds13** [comment]
I am also getting the same eror now

Launching lib\main.dart on moto e32 s in debug mode...
/C:/Users/admld/AppData/Local/Pub/Cache/hosted/pub.dev/velocity_x-4.2.0/lib/src/flutter/hover.dart:31:42: Error: The getter 'WidgetState' isn't defined for the class '_HoverToggleState'.
 - '_HoverToggleState' is from 'package:velocity_x/src/flutter/hover.dart' ('/C:/Users/admld/AppData/Local/Pub/Cache/hosted/pub.dev/velocity_x-4.2.0/lib/src/flutter/hover.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'WidgetState'.
        onEnter: (_) => setMaterialState(WidgetState.hovered, true),
                                         ^^^^^^^^^^^
/C:/Users/admld/AppData/Local/Pub/Cache/hosted/pub.dev/velocity_x-4.2.0/lib/src/flutter/hover.dart:32:41: Error: The getter 'WidgetState' isn't defined for the class '_HoverToggleState'.
 - '_HoverToggleState' is from 'package:velocity_x/src/flutter/hover.dart' ('/C:/Users/admld/AppData/Local/Pub/Cache/hosted/pub.dev/velocity_x-4.2.0/lib/src/flutter/hover.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'WidgetState'.
        onExit: (_) => setMaterialState(WidgetState.hovered, false),
                                        ^^^^^^^^^^^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'E:\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 5s
Error: Gradle task assembleDebug failed with exit code 1


Exited (1).

**Comment by 4RSIM3R** [comment]
facing same issue right here

**Comment by admlds13** [comment]
After updating the flutter and dependencies the problem resolved try to update flutter to the latest version some part of the flutter has also been change i noticed that as well

**Comment by iampawan** [comment]
Fixed with v4.2.1
